src/main.cpp

```diff
@@ -113,8 +115,9 @@ void usage()
   std::cerr << std::endl;
   std::cerr << "TROUBLESHOOTING OPTIONS:" << std::endl;
   std::cerr << "    -v                      verbose messages" << std::endl;
-  std::cerr << "    -d                      (dry run) debug info" << std::endl;
-  std::cerr << "    -dd                     (dry run) verbose debug info" << std::endl;
+  std::cerr << "    --dry-run               terminate execution right after attaching all the probes" << std::endl;
+  std::cerr << "    -d STAGE                (dry run) debug info for various stages of bpftrace execution" << std::endl;
```

Comment:
nit: maybe add the bit about how --dry-run is automatically added at the end of the text as it reads now it feels like -d means dry-run.

Reply:
Yeah, good idea, fixed that.